deploy/ui-frontend/frontend-nginxConfMap.yml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
```

Comment:
Is there a particular reason for this file to be a YAML? I think the data contained in here could replace/override the information in the existing /frontend/nginx/conf.d file. I have no problem if you want to use this and it works though.

Reply:
Yeah thats the reason to overload the current nginx file in order to match the system. Thought it was a cleaner way to have it setup this way rather than manually hardcoding it in the repo